postman/Fhir.postman_collection.json

```diff
@@ -124,11 +172,46 @@
 					"name": "PUT Binary PDF",
 					"request": {
 						"auth": {
-							"type": "bearer",
-							"bearer": [
+							"type": "oauth2",
+							"oauth2": [
+								{
+									"key": "redirect_uri",
+									"value": "http://localhost",
+									"type": "string"
+								},
+								{
+									"key": "state",
+									"value": "abc",
+									"type": "string"
+								},
+								{
+									"key": "scope",
+									"value": "profile openid",
+									"type": "string"
+								},
+								{
+									"key": "clientId",
+									"value": "{{CLIENT_ID}}",
+									"type": "string"
+								},
+								{
+									"key": "grant_type",
+									"value": "implicit",
+									"type": "string"
+								},
+								{
+									"key": "authUrl",
+									"value": "{{AUTH_URL}}",
```

Comment:
Same as above. Add docs for setting the `AUTH_URL` variable

Reply:
updated